app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,103 @@
  *= require_tree .
  *= require_self
  */
+html {
+    background-color: #343535b0;
+}
+
+.body-text {
+    background-color: rgb(209, 207, 207);
+    padding-bottom: 20px;
+    padding-top: 20px;
+}
+
+li {
+    list-style-type: none;
+    margin: 0 2vw;
+    font-size: 2vh;
+}
+
+nav {
+    width: 100vw;
+    background-color: #D6EFFF;
+}
+
+.header {
+    margin-left: 20px;
+    width: 100%;
+    text-decoration: none;
+    color: white;
+    padding: 2vw;
+    font-family: 'Courier New', Courier, monospace;
```

Comment:
Instead of having to specify the font-family every time you want to style some text you can add it once and it's applied globally. Generally you set this inside the body tag underneath the `html` tag. Like this:

```
  body {
    font-family: 'Courier New', Courier, monospace;
  }
```

Reply:
Hey @dylankwpe, you can set the things you've fixed to "done" by clicking on the "Resolve Conversation" button. This way we know what has been fixed and we can re-review 👍

Reply:
On it!